.github/workflows/CI.yml

```diff
@@ -442,6 +448,10 @@ jobs:
             --executable ./sunshine \
             --icon-file "../$ICON_FILE" \
             --desktop-file "./$DESKTOP_FILE" \
+            --library /usr/lib/x86_64-linux-gnu/dri/i965_drv_video.so \
+            --library /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so \
+            --library /usr/lib/x86_64-linux-gnu/dri/r600_drv_video.so \
+            --library /usr/lib/x86_64-linux-gnu/dri/radeonsi_drv_video.so \
```

Comment:
Maybe we could do something like this to solve #1792? (and any future similar issues)

```suggestion
            --library /usr/lib/x86_64-linux-gnu/dri/i965_drv_video.so \
            --library /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so \
            --library /usr/lib/x86_64-linux-gnu/dri/r600_drv_video.so \
            --library /usr/lib/x86_64-linux-gnu/dri/radeonsi_drv_video.so \
            ${extra_libraries} \
```

Then above this block of code somewhere:

```bash
# Get the list of libraries linked with the sunshine executable
libs=$(ldd ./sunshine | awk '{print $3}' | grep -v '^$')
extra_libraries=""
for lib in $libs; do
  extra_libraries+="--library $lib "
done
```